query-service-impl/src/test/java/org/hypertrace/core/query/service/QueryRequestBuilderUtils.java

```diff
@@ -94,6 +101,11 @@ public static Expression createAliasedFunctionExpression(
     return Filter.newBuilder().setOperator(operator).addAllChildFilter(Arrays.asList(childFilters));
   }
 
+  public static Filter createContainsKeyFilter(String column, List<String> values) {
```

Comment:
Tests can reference impl utils, no need to duplicate.

Reply:
resolved